src/main/java/io/pravega/connectors/flink/FlinkPravegaTableSource.java

```diff
@@ -35,7 +44,8 @@
  *
  * Supports both stream and batch environments.
  */
-public abstract class FlinkPravegaTableSource implements StreamTableSource<Row>, BatchTableSource<Row> {
+public abstract class FlinkPravegaTableSource implements StreamTableSource<Row>, BatchTableSource<Row>,
```

Comment:
Is it worth documenting this feature?

Reply:
The code has inline comments and I believe it should be suffice.